app/controllers/api/v1/shifts_controller.rb

```diff
@@ -0,0 +1,66 @@
+module Api
+  module V1
+    class ShiftsController < ApplicationController
+      # skip_before_action :verify_authenticity_token
+      before_action :set_shift, only: %i[ show update destroy ]
+
+      def index
+        shifts = Shift.all
+        render json: ShiftSerializer.new(shifts).serialized_json, status: :ok
+      end
+
+      def show
+        if @shift
+          render json: ShiftSerializer.new(@shift).serialized_json, status: :ok
+        else
+          render json: { error: "Unable to update shift" }, status: :unprocessable_entity
+        end
+      end
+
+      def create
+        user = User.find_by(id: shift_params[:user_id])
+        schedule = Schedule.find_by(id: shift_params[:schedule_id])
+        shift = Shift.new(
+          user: user,
+          schedule: schedule,
+          shift_start: shift_params[:shift_start],
+          shift_end: shift_params[:shift_end],
+        )
+
+        if shift.save
+          render json: { message: "Shift for #{shift.user.employee_name} created: #{shift.format_shift(shift.shift_start)} - #{shift.format_shift(shift.shift_end)}" }, status: :created
```

Comment:
Since you've already introduced the `Shift#format_shift` method, what do you think about creating a method that formats the shift time block output?  This would shorten the message string, isolate the business logic, and let you test what that looks like in a unit test instead of having to run a functional test.  It would also remove the need for passing any variables, since the shift start and end are both properties of the Shift?